src/graphql/resolvers/queries/community.js

```diff
@@ -26,43 +26,26 @@ export const fieldResolvers = {
     stats: async ({ slug }, __, { dataSources: { firestore } }) => {
       dlog('stats called %s', slug);
       if (!slug) return [];
-
+      const today = new Date();
       const allEvents = await eventStore(firestore).findAllByCommunitySlug(
         slug,
       );
-      const sessions = await sessionStore(
+      const totalMembers = await memberStore(firestore).getMemberTotal();
+      const stats = await sessionStore(
         firestore,
-      ).findAllAcceptedByEventIdBatch(allEvents.map(e => e.id));
```

Comment:
The current change is only a slight improvement in performance, though with many fewer queries. Removing this requirement (find by event id) the sessions are now queried once instead of by event. 

Total query count for this refactoring is 3 (members, events